README.md

```diff
@@ -1,41 +1,38 @@
-# ChugSplash
+# ChugSplash Foundry
 
-A Hardhat and Foundry plugin for deploying and managing upgradeable smart contracts.
-
-If you're using Foundry, check out out our Foundry plugin [here](https://github.com/chugsplash/chugsplash-foundry). Otherwise, keep reading for the Hardhat plugin.
+A modern, declarative, and deterministic smart contract deployment and upgrade framework for Foundry and Hardhat.
```

Comment:
Thoughts on leaving 'modern' out? It isn't very descriptive and sounds a little buzzwordy IMO

Also, I think a slightly better way to rephrase this would be "A declarative and deterministic framework for deploying and upgrading smart contracts. Available as a Foundry and Hardhat plugin."

Reply:
Yeah, I'm fine with that description. I don't think modern is a superfluous word though. I think it implies this is a tool that is on par with the quality of modern web2 developer tools.